examples/examples-by-ml-library/libraries/util/datasets.py

```diff
@@ -50,6 +51,23 @@ def load_regression_dataframe():
     return df
 
 
+def load_causal_regression_dataset():
```

Comment:
Not sure if there are other places this might be useful, but was having to copy this multiple times just for the causal ML examples so added a helper function for it 

The "treatment" is entirely dummy / fake 

Future enhancement might be to source a proper dataset for causal modelling (although I couldn't easily find one that was easily accessible without adding data files to this repo)